content/news/027/index.md

```diff
@@ -114,6 +114,23 @@ The code source is freely [available on GitHub](https://github.com/djeedai/libra
 [@djeedai]: https://twitter.com/djeedai
 [Bevy Engine]: https://bevyengine.org/
 
+### [Berry Run] - !!! PLAY IN YOUR BROSWER NOW !!!
+
+![berry](berrycover.png)
+
+[Berry Run] is a bad game by [@bombfuse_dev] built on top of
+[Emerald Engine], follow them on twitter, they're cool.
```

Comment:
Can this be rephrased to be a bit more descriptive of what the game is/a little less self-deprecating? For people who don't know the context of how the newsletter is written, it'd look the editorial team were being very rude to you 😆

Reply:
Haha yah I can definitely fix this up, I didn't really think about how it might sound like it's coming from someone else haha

Reply:
Looks good now :)